content/docs/command-reference/add.md

```diff
@@ -243,3 +243,39 @@ In this case, a DVC-file is generated for each file in the `pics/` directory
 tree. No top-level DVC-file is generated, which is typically less convenient.
 For example, we cannot use the directory structure as one unit with `dvc run` or
 other commands.
+
+## Example: Dvcignore
+
+To untrack a file or directory just add
+[patterns](https://git-scm.com/docs/gitignore) associated with that file or
+directory under `.dvcignore` file.
+
+Let's take an example to illustrate this.
+
+```dvc
+$ mkdir Dir
+$ echo file_one > Dir/file1
+$ echo file_two > Dir/file2
+```
+
+Now add `file1` to `.dvcignore` and track `Dir` directory with `dvc add`.
+
+```dvc
+$ echo Dir/file1 > .dvcignore
+$ dvc add Dir
+$ cat .dvcignore
+Dir/file1
+```
+
+Let's now modify the file which we added to `.dvcignore` and run `dvc status`.
+
+```dvc
+$ echo file_one_changed > Dir/file1
```

Comment:
```suggestion
$ mkdir dir
$ echo file_one > dir/file1
$ echo file_two > dir/file2
```

Now add `file1` to `.dvcignore` and track `dir` directory with `dvc add`.

```dvc
$ echo dir/file1 > .dvcignore
$ dvc add dir
$ cat .dvcignore
dir/file1
```

Let's now modify the file which we added to `.dvcignore` and run `dvc status`.

```dvc
$ echo file_one_changed > dir/file1
```